### PR TITLE
Ignore spec files to allow for co-located tests

### PR DESCRIPTION
**Pull request by okcoker**
Instead of having all my tests under `__tests__`, I wanted to co-locate them alongside my src files (ie `actions/status.ts` and `actions/status.test.ts`). I had a passing test suite when I had everything under `__tests__` but after moving to a co-located folder structure, my test suite failed with cryptic errors.

This seems to fix that.

### CODE REVIEW

File: __tests__/modules/utils/ensureNoTsHeaderOrSpecFiles.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  });
+
+  it("filters test/spec files", () => {
+    const filtered = ensureNoTsHeaderOrSpecFiles([
+      "afdasf.spec.ts",
+      "afdasfsdfa.spec.js",
+      "afdasfsdfa.test.js",
+      "afdasfsdfa.test.ts",
+      "some-file.ts",
+    ]);
```
Comment (evantahler) [inline __tests__/modules/utils/ensureNoTsHeaderOrSpecFiles.ts:19]: Can you please add one more test case asserting that `test-my-api.js` or something like that is still OK (that `.test.` is what is ignored).

**Review by evantahler — CHANGES_REQUESTED** [review]

File: __tests__/modules/utils/ensureNoTsHeaderOrSpecFiles.ts (hunk 1 above)
Comment (evantahler) [reply __tests__/modules/utils/ensureNoTsHeaderOrSpecFiles.ts:19]: Otherwise, this PR is 👍!

Comment (okcoker) [reply __tests__/modules/utils/ensureNoTsHeaderOrSpecFiles.ts]: Done!

**Review by evantahler — APPROVED** [review]
🥳

**Comment by evantahler** [comment]
released! https://github.com/actionhero/actionhero/releases/tag/v28.2.0

**Pull request merged**